code/sonalyze/db/sample_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	. "sonalyze/common"
 )
 
-func TestParseSonarLogTagged(t *testing.T) {
+func TestParseCSVSonarLogTagged(t *testing.T) {
```

Comment:
Probably "TestParseSampleCSV" or something, according to the FileAttr name.  Also below.